crates/biome_lsp/src/handlers/formatting.rs

```diff
@@ -61,17 +64,13 @@ pub(crate) fn format(
             _ => {}
         }
 
-        let content = session.workspace.get_file_content(GetFileContentParams {
-            path: biome_path.clone(),
-        })?;
```

Comment:
This handler would retrieve the file contents twice, which seemed unnecessary. In theory, it could even lead to race conditions, so it made me think about the bug we used to have when multiple processes were registered in VS Code, which would cause the formatter to mess up some lines...